x-pack/plugins/security_solution/public/entity_analytics/api/hooks/use_preview_risk_scores.ts

```diff
@@ -9,38 +9,50 @@ import dateMath from '@kbn/datemath';
 import type { RiskScoresPreviewRequest } from '../../../../common/api/entity_analytics/risk_engine/preview_route.gen';
 import { useEntityAnalyticsRoutes } from '../api';
 
+export type UseRiskScorePreviewParams = Omit<RiskScoresPreviewRequest, 'data_view_id'> & {
```

Comment:
I don't really understand why we need to do this?

The only reason would be to make the `data_view_id` field optional, but why not just edit the original type rather than wrapping it in an Omit + intersection?